### DISCUSSION TITLE
Não Trapaceie um Trapaceiro

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#482] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/nao-trapaceie-um-trapaceiro/

**Comment by one-eyed-willy** [comment]
Profissional não vale :grin:

**Comment by angellisy** [comment]
ATLA é simplesmente fantástico :heart\_eyes:

**Comment by maryon-telvanni** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/videos/nao-trapaceie-um-trapaceiro/#comment_1160697560>):

> ATLA é simplesmente fantástico 😍

Queria assistir "A Lenda de Korra". Será que é bom?

**Comment by one-eyed-willy** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/videos/nao-trapaceie-um-trapaceiro/#comment_1160732817>):

> Queria assistir "A Lenda de Korra". Será que é bom?

A pegada é um pouco diferente, mais pode assistir
